contracts/strategy/InvestorStrategy.sol

```diff
@@ -2,37 +2,83 @@
 // Copyright Fathom 2023
 pragma solidity 0.8.19;
 
-import "./BaseStrategy.sol";
+import "./TokenizedStrategy.sol";
 import "./interfaces/IInvestor.sol";
 import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 // solhint-disable
-contract InvestorStrategy is BaseStrategy {
+contract InvestorStrategy is TokenizedStrategy {
     using SafeERC20 for ERC20;
 
     IInvestor public immutable investor;
 
-    constructor(address _investor, address _asset, string memory _name) BaseStrategy(_asset, _name) {
+    uint256 public minDebt;
+    uint256 public maxDebt = type(uint256).max;
+
+    /// @notice Private variables and functions used in this mock.
+    bytes32 public constant BASE_STRATEGY_STORAGE = bytes32(uint256(keccak256("fathom.base.strategy.storage")) - 1);
```

Comment:
Duplicate in TokenizedStrategy

Reply:
I had undo that change and adjusted code accordingly to previous version